delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/Digest.scala

```diff
@@ -21,4 +23,9 @@ object Digest {
     * A digest that does not yield a value because it is still being computed
     */
   final case object NotComputedDigest extends Digest
+
+  implicit val digestEncoder: Encoder.AsObject[Digest] = Encoder.encodeJsonObject.contramapObject {
+    case ComputedDigest(algorithm, value) => JsonObject("_algorithm" -> algorithm.asJson, "_value" -> value.asJson)
+    case NotComputedDigest                => JsonObject("_value" -> "".asJson)
```

Comment:
Shouldn't it be JNull ? Unless it was implemented this way in v1.4 ?

Reply:
It was indeed like this in 1.4.x. It returned a value as an empty string